src/c++/perf_analyzer/genai-pa/genai_pa/llm_inputs/llm_inputs.py

```diff
@@ -37,54 +57,105 @@ class LlmInputs:
     DEFAULT_LENGTH = 100
     MINIMUM_LENGTH = 1
 
+    EMPTY_JSON_IN_VLLM_PA_FORMAT = {"data": []}
+    EMPTY_JSON_IN_TRTLLM_PA_FORMAT = {"data": []}
```

Comment:
Is it necessary to have both here if they start the same way?

Reply:
Necessary: no. Good as a form of documentation: yes.
My preference is to be verbose and to show that thought was put into creating the defines and to future proof in case things ever change.

Reply:
thats fine by me.
it was done with intention, and not copy paste mistakes.